docs/checkit.js

```diff
@@ -711,26 +709,27 @@ CheckIt.prototype.deleteCalendar = function() {
         // if rejected promise returned catch gets run
         .catch(function(err) {
             console.error("Couldn't get active calendar id: " + err);
-            // we don't return a resolved promise here, so after this, 
-            // we exit the function?
+            // Notify user that there is no calendar on display for them
+            // to delete? Should I bother notifying them of that?
+            $.notify("No Calendar On Display");
+            // exit function, does returning err cause us to exit the function?
+            // or do I need to return a rejected promise?
+            return err;
```

Comment:
Returning err will cause the next `.then()` in the chain to run with `err` as its argument.